index.html

```diff
@@ -2,9 +2,12 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Моя первая страница</title>
 </head>
 <body>
-
+<strong>Меня зовут Алена</strong> и &#128155 я всю жизнь рисую. &#128567 Я училась на преподавателя английского языка <p>не с целью преподавания</p> <br>но &euro потому что изучение английского было также моим любимым занятием. <i>Я выбрала фронтэнд поскольку мне нравится идея того, что я могу работать из любой точки света.</i> <p>Так же я получаю удовольствие <h2>когда пытаюсь разобраться как веб работает изнутри</h2> поскольку <em>я визуал</em> <p>и всегда любила наблюдать преимущественно результат.</p> <small>Я люблю читать книги, смотреть фильмы и проводть время с хорошими людьми.</small> <hr> <br>Из объективных минусов: я в последствии страдаю что в режиме такой жизни ты <h3>как правило не можешь сохранять близких людей около себя все время.</h3> С другой же стороны это позволяет полноценно и в моменте жить жизнь и стараться не застревать на одном<sub>2</sub>. <mark>Начав жить такую сумбурную жизнь переездов в раннем возресте</mark> <del>я так же рано для себя поняла</del> <ins>что мне необходима на самом то деле стабильность</ins> и устойчивость<sup>2</sup>. На что я сейчас и зарабатываю деньги. <h4>Ну и конечно правильных людей рядом:)</h4>
```

Comment:
you need to refactor this code: new tag -> new string / now it`s realy hard to read